Create ui-c8y-1024-9-0-support-download-link-in-login-app-12365.md (#4960)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1024-9-0-support-download-link-in-login-app-12365.md

---
date: ""

> title: Support download links in Login app

Reviewer: ```suggestion
title: Support download links in login app
```

product_area: Platform services
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-67167
version: 1024.9.0
---

> The login app now supports direct file downloads through shareable links, allowing you to share download links (for example, `https://jsmith.preprod.c8y.io/apps/public/login/index.html#/?download=78180895`) without requiring a custom application. Previously, the login app could only redirect you to your default application after authentication. Now when you open a download link, the login app authenticates you normally (including single sign-on), then displays an inline download view showing the file name, a download button, and the app switcher instead of redirecting to your default app. After you download the file, a success message appears and you can navigate to any app using the switcher or log out. This simplifies sharing files with users by eliminating the need to create custom applications for download functionality.

Reviewer: ```suggestion
The login app now supports direct file downloads through shareable links, allowing you to share download links (for example, *https://jsmith.preprod.c8y.io/apps/public/login/index.html#/?download=78180895*) without requiring a custom application. Previously, the login app could only redirect you to your default application after authentication. Now, when you open a download link, the login app authenticates you as usual (including single sign-on), then displays an inline download view showing the file name, a download button, and the app switcher instead of redirecting to your default app. After you download the file, a success message appears, and you can navigate to any app using the switcher or log out. This simplifies sharing files with users by eliminating the need to create custom applications for download functionality.
```

Reviewer: I´m not sure if "login app" is an official name that we share with customers? @janhommes @rahulc8y

---
date: ""

> title: Support download link in Login app (#12365)

Reviewer 2: ```suggestion
title: Support download links in Login app
```
